test/src/e2e.dynval/dv.double-vote.test.ts

```diff
@@ -17,15 +17,9 @@
 import * as chai from "chai";
 import { expect } from "chai";
 import * as chaiAsPromised from "chai-as-promised";
+import { SDK } from "codechain-sdk";
 import * as stake from "codechain-stakeholder-sdk";
-import "mocha";
-import * as RLP from "rlp";
-
-import { validators as originalValidators } from "../../tendermint.dynval/constants";
```

Comment:
Apply the changes in this commit to the previous one if possible.
We don't need to make a separated commit if we can change history.

Reply:
I applied the changes in this commit to each commit before, and deleted it.